core/domain/general_feedback_services.py

```
@@ -320,30 +321,52 @@ def get_lesson_feedback_summaries(
         if date_to_msecs is not None
         else None
     )
+    filter = [status_filter] if status_filter else None
```

Ruff (0.16.1)

[error] Variable `filter` is shadowing a Python builtin

(A001)